packages/app/src/components/VisualizationOptions/styles/VisualizationOptions.style.js

```diff
@@ -1,26 +1,26 @@
-import React from 'react';
-import css from 'styled-jsx/css';
+import React from 'react' /* eslint-disable-line no-unused-vars */
```

Comment:
why disable the rule?

Reply:
See: https://github.com/dhis2/app-platform/issues/213

Reply:
This is most probably temporary, since it seems for apps we are going to use CSS modules.
But I'm waiting for the final decision on that.